spec/bookingsync/api/client/rentals_spec.rb

```diff
@@ -65,6 +65,16 @@
     it "returns a single rental" do
       rental = client.rental(prefetched_rental[:id])
       expect(rental.name).to eql(prefetched_rental[:name])
+      expect(rental).to_not have_key(:availability)
+    end
+
+    context "with additional query params" do
+      it "returns a single rental with defined params" do
+        rental = client.rental(prefetched_rental[:id], include: [:availability, :change_over])
+        expect(rental.name).to eql(prefetched_rental[:name])
```

Comment:
I assumed it was there for some reason before (https://github.com/BookingSync/bookingsync-api/pull/105/files#diff-ad09b46d8f1cdfbf25b51538edf54dc1L67), so I decided to not change it. Maybe it was vital to check if types are matching as well, I'm not sure.
And I see eql is actually used in more places, so if we want to be consistent, it may be worth changing it everywhere.